### DISCUSSION TITLE
OBS 30 MacOS incompatibility

### DISCUSSION THREAD

**Discussion by WarmUpTill** [WarmUpTill/SceneSwitcher#829] [Announcements]
### If you are *not* using MacOS, you can stop reading here!

If you are running the OBS 30 beta releases on MacOS, the official releases of the Advanced Scene Switcher plugin will not function. In fact, having the plugin installed will result in OBS immediately crashing on startup and prompting you to start in "Safe mode" without any plugins!

Other platforms are *not* affected!

### What do I need to do?

The first step should be to [back up](https://github.com/WarmUpTill/SceneSwitcher/wiki/Saving-and-loading-settings#how-to-create-and-import-a-backup-of-your-settings) your existing settings while using the previously compatible version of OBS again.
You will have to do this for each scene collection.

If you want to continue using the Advanced Scene Switcher plugin with the OBS 30 beta, you will have to use this development release of the plugin:

https://github.com/WarmUpTill/SceneSwitcher/actions/runs/5741802989?pr=800

Note that you will have to be logged into GitHub to be able to download it.
Also note that this version of the plugin is *not* compatible with OBS versions older than OBS 30.

### Why is this happening?

This is happening due to an [incompatibility](https://github.com/chriskohlhoff/asio/issues/975) in the asio library, which was updated to version 1.28.0 with OBS 30.
The plugin will switch to this new version once OBS 30 was released officially.

**Comment by MathiasWolters** [comment]
How to download this extension for mac. There is no download link when you open the page: https://github.com/WarmUpTill/SceneSwitcher/actions/runs/5741802989?pr=800
I have no idea how I can get downloaded this advanced scene switcher for mac.
Thanks for help!

**Reply by WarmUpTill** [reply]
This announcement created when the OBS 30 beta was released and the link contained a pre-release compatible with those beta releases.
In the meantime OBS 30 has long been released and you can install the regular releases of the plugin assuming you have at least OBS 30 installed.

![Install](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/0abff540-5a6a-45da-9239-7877464fae80)

If you are stuck on an older version of OBS you will have to stick with this version of the plugin:
https://github.com/WarmUpTill/SceneSwitcher/releases/tag/1.23.1

I will remove the compatibility note from the release notes to avoid this confusion in the future.
Sorry about that!

**Reply by MathiasWolters** [reply]
Thank you so much. Working great!
